examples/tts/conf/es/fastpitch_align_44100_ipa.yaml

```diff
@@ -13,9 +13,10 @@ sup_data_types: ["align_prior_matrix", "pitch"]
 pitch_fmin: 65.40639132514966
 pitch_fmax: 2093.004522404789
 
-# Stats (per frame), train (these values depend on pitch_fmin and pitch_fmax)
-pitch_mean: 212.35873413085938
-pitch_std: 68.52806091308594
+# these frame-wise values depend on pitch_fmin and pitch_fmax, you can get values
+# by running `scripts/dataset_processing/tts/extract_sup_data.py`
+pitch_mean: ???  # e.g. 212.35873413085938
+pitch_std:  ???  # e.g.  68.52806091308594
```

Comment:
@rlangman please let me know what dataset these values were computed on. Then I will update here. Thanks.

Reply:
yeah, talked with @rlangman offline. Revised in the new commit.